devnet/docker-compose-substrate.yml

```
@@ -0,0 +1,58 @@
+services:
+  prometheus:
+    image: prom/prometheus
+    volumes:
+    - ./../prometheus.yml:/etc/prometheus/prometheus.yml
```

nit maybe read only?

```suggestion
    - ./../prometheus.yml:/etc/prometheus/prometheus.yml:ro
```